recipes/foxglove-websocket/all/conanfile.py

```diff
@@ -20,10 +20,12 @@ class FoxgloveWebSocketConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "asio": ["boost", "standalone", False],
```

Comment:
This is adding an option that is not going to be tested in ci. Did you actually try to build this with other option besides `standalone`? Can you provide a successful build log for that?

Reply:
I tried building this with `boost` and that works correctly, however, it failed for `False`. I'm not having a lot of time looking into this, so I will just remove the `False` option.
Do I even need to specify this option here which is basically just forwarded to websocketpp? I guess a recipe user could also just pass `-o websocketpp:asio=boost` ?

<details><summary>Build logs for -o foxglove-websocket:asio=boost</summary><pre>achim@achim-pc:~/workspace/conan-center-index/recipes/foxglove-websocket$ conan create all/conanfile.py foxglove-websocket/1.1.0@ -pr:b=default -pr:h=default -o foxglove-
websocket:asio=boost
[HOOK - conan-center.py] pre_export(): [DEPRECATED GLOBAL CPPSTD (KB-H001)] OK
[HOOK - conan-center.py] pre_export(): [REFERENCE LOWERCASE (KB-H002)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE METADATA (KB-H003)] OK
[HOOK - conan-center.py] pre_export(): [HEADER_ONLY, NO COPY SOURCE (KB-H005)] OK
[HOOK - conan-center.py] pre_export(): [FPIC OPTION (KB-H006)] OK
[HOOK - conan-center.py] pre_export(): [VERSION RANGES (KB-H008)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] Total recipe size: 6.1044921875 KB
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] OK
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: None
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: None
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE FOLDER (KB-H024)] OK
[HOOK - conan-center.py] pre_export(): [META LINES (KB-H025)] OK
[HOOK - conan-center.py] pre_export(): [CONAN CENTER INDEX URL (KB-H027)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE MINIMUM VERSION (KB-H028)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - RUN ENVIRONMENT (KB-H029)] OK
[HOOK - conan-center.py] pre_export(): [SYSTEM REQUIREMENTS (KB-H032)] OK
[HOOK - conan-center.py] pre_export(): [CONANDATA.YML FORMAT (KB-H030)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - NO IMPORTS() (KB-H034)] OK
[HOOK - conan-center.py] pre_export(): [NO AUTHOR (KB-H037)] OK
[HOOK - conan-center.py] pre_export(): [NOT ALLOWED ATTRIBUTES (KB-H039)] OK
[HOOK - conan-center.py] pre_export(): [NO TARGET NAME (KB-H040)] OK
[HOOK - conan-center.py] pre_export(): [NO FINAL ENDLINE (KB-H041)] OK
[HOOK - conan-center.py] pre_export(): [NO REQUIRES.ADD() (KB-H044)] OK
[HOOK - conan-center.py] pre_export(): [DELETE OPTIONS (KB-H045)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERBOSE MAKEFILE (KB-H046)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERSION REQUIRED (KB-H048)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE WINDOWS EXPORT ALL SYMBOLS (KB-H049)] OK
[HOOK - conan-center.py] pre_export(): [DEFAULT OPTIONS AS DICTIONARY (KB-H051)] OK
[HOOK - conan-center.py] pre_export(): [CONFIG.YML HAS NEW VERSION (KB-H052)] OK
[HOOK - conan-center.py] pre_export(): [PRIVATE IMPORTS (KB-H053)] OK
[HOOK - conan-center.py] pre_export(): [SINGLE REQUIRES (KB-H055)] OK
[HOOK - conan-center.py] pre_export(): [TOOLS RENAME (KB-H057)] OK
[HOOK - conan-center.py] pre_export(): [ILLEGAL CHARACTERS (KB-H058)] OK
[HOOK - conan-center.py] pre_export(): [CLASS NAME (KB-H059)] OK
[HOOK - conan-center.py] pre_export(): [NO CRLF (KB-H060)] OK
[HOOK - conan-center.py] pre_export(): [NO BUILD SYSTEM FUNCTIONS (KB-H061)] OK
[HOOK - conan-center.py] pre_export(): [TOOLS CROSS BUILDING (KB-H062)] OK
[HOOK - conan-center.py] pre_export(): [INVALID TOPICS (KB-H064)] OK
[HOOK - conan-center.py] pre_export(): [NO REQUIRED_CONAN_VERSION (KB-H065)] OK
[HOOK - conan-center.py] pre_export(): [TEST_TYPE MANAGEMENT (KB-H068)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - NO DEFAULT OPTIONS (KB-H069)] OK
[HOOK - conan-center.py] pre_export(): [MANDATORY SETTINGS (KB-H070)] OK
[HOOK - conan-center.py] pre_export(): [PYLINT EXECUTION (KB-H072)] OK
[HOOK - conan-center.py] pre_export(): [TEST V1 PACKAGE FOLDER (KB-H073)] OK
[HOOK - conan-center.py] pre_export(): [REQUIREMENT OVERRIDE PARAMETER (KB-H075)] OK
Exporting package recipe
foxglove-websocket/1.1.0 exports: File 'conandata.yml' found. Exporting it...
foxglove-websocket/1.1.0 exports: Copied 1 '.yml' file: conandata.yml
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] Saving conandata.yml: {'sources': {'1.1.0': {'url': 'https://github.com/foxglove/ws-protocol/archive/refs/tags/releases/cpp/v1.1.0.tar.gz', 'sha256': '360c246f5edccbe2a7c905ff77c5fb2b8ca723e6b441613871952d9d19903362'}}}
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] New conandata.yml contents: sources:
  1.1.0:
    sha256: 360c246f5edccbe2a7c905ff77c5fb2b8ca723e6b441613871952d9d19903362
    url: https://github.com/foxglove/ws-protocol/archive/refs/tags/releases/cpp/v1.1.0.tar.gz

[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] OK
[HOOK - conan-center.py] post_export(): [DEFAULT SHARED OPTION VALUE (KB-H050)] OK
[HOOK - conan-center.py] post_export(): [LICENSE PUBLIC DOMAIN (KB-H056)] OK
foxglove-websocket/1.1.0: The stored package has not changed
foxglove-websocket/1.1.0: Exported revision: 067f8793cbc242cc56a383a86713f311
Configuration (profile_host):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
foxglove-websocket:asio=boost
[build_requires]
[env]

Configuration (profile_build):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

foxglove-websocket/1.1.0: Forced build from source
Version ranges solved
    Version range '>=1.1 <4' required by 'websocketpp/0.8.2' resolved to 'openssl/3.1.0' in local cache

foxglove-websocket/1.1.0 (test package): Installing package
Requirements
    boost/1.81.0 from 'conancenter' - Cache
    bzip2/1.0.8 from 'conancenter' - Cache
    foxglove-websocket/1.1.0 from local cache - Cache
    libbacktrace/cci.20210118 from 'conancenter' - Cache
    nlohmann_json/3.10.5 from 'conancenter' - Cache
    openssl/3.1.0 from 'conancenter' - Cache
    websocketpp/0.8.2 from 'conancenter' - Cache
    zlib/1.2.13 from 'conancenter' - Cache
Packages
    boost/1.81.0:9bb382e9c85821936d59f1c4ea5134768df38de2 - Cache
    bzip2/1.0.8:c32092bf4d4bb47cf962af898e02823f499b017e - Cache
    foxglove-websocket/1.1.0:b9ec97f499de3078e925225c508a1962fdfe3df4 - Build
    libbacktrace/cci.20210118:dfbe50feef7f3c6223a476cd5aeadb687084a646 - Cache
    nlohmann_json/3.10.5:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    openssl/3.1.0:304480252b01879c8641f79a653b593b8f26cf9f - Cache
    websocketpp/0.8.2:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    zlib/1.2.13:dfbe50feef7f3c6223a476cd5aeadb687084a646 - Cache

Installing (downloading, building) binaries...
bzip2/1.0.8: Already installed!
libbacktrace/cci.20210118: Already installed!
nlohmann_json/3.10.5: Already installed!
zlib/1.2.13: Already installed!
boost/1.81.0: Already installed!
openssl/3.1.0: Already installed!
websocketpp/0.8.2: Already installed!
foxglove-websocket/1.1.0: Copying sources to build folder
foxglove-websocket/1.1.0: Building your package in /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4
foxglove-websocket/1.1.0: Generator 'CMakeDeps' calling 'generate()'
foxglove-websocket/1.1.0: Calling generate()
foxglove-websocket/1.1.0: Preset 'release' added to CMakePresets.json. Invoke it manually using 'cmake --preset release'
foxglove-websocket/1.1.0: If your CMake version is not compatible with CMakePresets (<3.19) call cmake like: 'cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release/generators/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release'
foxglove-websocket/1.1.0: Aggregating env generators
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK. 'fPIC' option found and apparently well managed
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK
foxglove-websocket/1.1.0: Calling build()
foxglove-websocket/1.1.0: CMake command: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/src"
-- Using Conan toolchain: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The CXX compiler identification is GNU 11.3.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Target declared 'nlohmann_json::nlohmann_json'
-- Conan: Target declared 'websocketpp::websocketpp'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/achim/.conan/data/openssl/3.1.0/_/_/package/304480252b01879c8641f79a653b593b8f26cf9f/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Component target declared 'Boost::diagnostic_definitions'
-- Conan: Component target declared 'Boost::disable_autolinking'
-- Conan: Component target declared 'Boost::dynamic_linking'
-- Conan: Component target declared 'Boost::headers'
-- Conan: Component target declared 'Boost::boost'
-- Conan: Component target declared 'boost::_libboost'
-- Conan: Component target declared 'Boost::atomic'
-- Conan: Component target declared 'Boost::container'
-- Conan: Component target declared 'Boost::context'
-- Conan: Component target declared 'Boost::date_time'
-- Conan: Component target declared 'Boost::exception'
-- Conan: Component target declared 'Boost::math'
-- Conan: Component target declared 'Boost::math_c99'
-- Conan: Component target declared 'Boost::math_c99f'
-- Conan: Component target declared 'Boost::math_c99l'
-- Conan: Component target declared 'Boost::math_tr1'
-- Conan: Component target declared 'Boost::math_tr1f'
-- Conan: Component target declared 'Boost::math_tr1l'
-- Conan: Component target declared 'Boost::program_options'
-- Conan: Component target declared 'Boost::regex'
-- Conan: Component target declared 'Boost::serialization'
-- Conan: Component target declared 'Boost::stacktrace'
-- Conan: Component target declared 'Boost::stacktrace_addr2line'
-- Conan: Component target declared 'Boost::stacktrace_backtrace'
-- Conan: Component target declared 'Boost::stacktrace_basic'
-- Conan: Component target declared 'Boost::stacktrace_noop'
-- Conan: Component target declared 'Boost::system'
-- Conan: Component target declared 'Boost::test'
-- Conan: Component target declared 'Boost::test_exec_monitor'
-- Conan: Component target declared 'Boost::url'
-- Conan: Component target declared 'Boost::wserialization'
-- Conan: Component target declared 'Boost::chrono'
-- Conan: Component target declared 'Boost::coroutine'
-- Conan: Component target declared 'Boost::filesystem'
-- Conan: Component target declared 'Boost::json'
-- Conan: Component target declared 'Boost::nowide'
-- Conan: Component target declared 'Boost::prg_exec_monitor'
-- Conan: Component target declared 'Boost::random'
-- Conan: Component target declared 'Boost::thread'
-- Conan: Component target declared 'Boost::timer'
-- Conan: Component target declared 'Boost::type_erasure'
-- Conan: Component target declared 'Boost::unit_test_framework'
-- Conan: Component target declared 'Boost::wave'
-- Conan: Component target declared 'Boost::contract'
-- Conan: Component target declared 'Boost::fiber'
-- Conan: Component target declared 'Boost::fiber_numa'
-- Conan: Component target declared 'Boost::graph'
-- Conan: Component target declared 'Boost::iostreams'
-- Conan: Component target declared 'Boost::locale'
-- Conan: Component target declared 'Boost::log'
-- Conan: Component target declared 'Boost::log_setup'
-- Conan: Target declared 'boost::boost'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/home/achim/.conan/data/bzip2/1.0.8/_/_/package/c32092bf4d4bb47cf962af898e02823f499b017e/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'libbacktrace::libbacktrace'
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_POLICY_DEFAULT_CMP0091


-- Build files have been written to: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release
foxglove-websocket/1.1.0: CMake command: cmake --build "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release" '--' '-j16'
[ 50%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/parameter.cpp.o
[ 50%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/base64.cpp.o
[ 75%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/serialization.cpp.o
[100%] Linking CXX static library libfoxglove_websocket.a
[100%] Built target foxglove_websocket
foxglove-websocket/1.1.0: Package 'b9ec97f499de3078e925225c508a1962fdfe3df4' built
foxglove-websocket/1.1.0: Build folder /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release
foxglove-websocket/1.1.0: Generated conaninfo.txt
foxglove-websocket/1.1.0: Generated conanbuildinfo.txt
foxglove-websocket/1.1.0: Generating the package
foxglove-websocket/1.1.0: Package folder /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4
foxglove-websocket/1.1.0: Calling package()
foxglove-websocket/1.1.0: CMake command: cmake --install "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/b9ec97f499de3078e925225c508a1962fdfe3df4/build/Release" --prefix "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4"
-- Install configuration: "Release"
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/lib/libfoxglove_websocket.a
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/base64.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/server_interface.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/websocket_tls.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/websocket_client.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/websocket_logging.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/websocket_server.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/callback_queue.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/regex_utils.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/websocket_notls.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/serialization.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/parameter.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/include/foxglove/websocket/common.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/b9ec97f499de3078e925225c508a1962fdfe3df4/licenses/LICENSE
[HOOK - conan-center.py] post_package(): [PACKAGE LICENSE (KB-H012)] OK
[HOOK - conan-center.py] post_package(): [DEFAULT PACKAGE LAYOUT (KB-H013)] OK
[HOOK - conan-center.py] post_package(): [MATCHING CONFIGURATION (KB-H014)] OK
[HOOK - conan-center.py] post_package(): [SHARED ARTIFACTS (KB-H015)] OK
[HOOK - conan-center.py] post_package(): [STATIC ARTIFACTS (KB-H074)] OK
[HOOK - conan-center.py] post_package(): [EITHER STATIC OR SHARED OF EACH LIB (KB-H076)] OK
[HOOK - conan-center.py] post_package(): [PC-FILES (KB-H020)] OK
[HOOK - conan-center.py] post_package(): [CMAKE-MODULES-CONFIG-FILES (KB-H016)] OK
[HOOK - conan-center.py] post_package(): [PDB FILES NOT ALLOWED (KB-H017)] OK
[HOOK - conan-center.py] post_package(): [LIBTOOL FILES PRESENCE (KB-H018)] OK
[HOOK - conan-center.py] post_package(): [MS RUNTIME FILES (KB-H021)] OK
[HOOK - conan-center.py] post_package(): [SHORT_PATHS USAGE (KB-H066)] OK
[HOOK - conan-center.py] post_package(): [MISSING SYSTEM LIBS (KB-H043)] OK
[HOOK - conan-center.py] post_package(): [APPLE RELOCATABLE SHARED LIBS (KB-H077)] OK
foxglove-websocket/1.1.0 package(): Packaged 1 file: LICENSE
foxglove-websocket/1.1.0 package(): Packaged 1 '.a' file: libfoxglove_websocket.a
foxglove-websocket/1.1.0 package(): Packaged 12 '.hpp' files
foxglove-websocket/1.1.0: Package 'b9ec97f499de3078e925225c508a1962fdfe3df4' created
foxglove-websocket/1.1.0: Created package revision 72d531bf72a0a31d8577e4d460d3f35e
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
[HOOK - conan-center.py] post_package_info(): [LIBRARY DOES NOT EXIST (KB-H054)] OK
[HOOK - conan-center.py] post_package_info(): [INCLUDE PATH DOES NOT EXIST (KB-H071)] OK
foxglove-websocket/1.1.0 (test package): Generator 'CMakeDeps' calling 'generate()'
foxglove-websocket/1.1.0 (test package): Generator txt created conanbuildinfo.txt
foxglove-websocket/1.1.0 (test package): Calling generate()
foxglove-websocket/1.1.0 (test package): Preset 'release' added to CMakePresets.json. Invoke it manually using 'cmake --preset release'
foxglove-websocket/1.1.0 (test package): If your CMake version is not compatible with CMakePresets (<3.19) call cmake like: 'cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release'
foxglove-websocket/1.1.0 (test package): Aggregating env generators
foxglove-websocket/1.1.0 (test package): Generated conaninfo.txt
foxglove-websocket/1.1.0 (test package): Generated graphinfo
Using lockfile: '/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan.lock'
Using cached profile from lockfile
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not found
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK
foxglove-websocket/1.1.0 (test package): Calling build()
foxglove-websocket/1.1.0 (test package): CMake command: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/."
-- Using Conan toolchain: /home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake
-- Conan: Target declared 'foxglove-websocket::foxglove-websocket'
-- Conan: Target declared 'nlohmann_json::nlohmann_json'
-- Conan: Target declared 'websocketpp::websocketpp'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/achim/.conan/data/openssl/3.1.0/_/_/package/304480252b01879c8641f79a653b593b8f26cf9f/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Component target declared 'Boost::diagnostic_definitions'
-- Conan: Component target declared 'Boost::disable_autolinking'
-- Conan: Component target declared 'Boost::dynamic_linking'
-- Conan: Component target declared 'Boost::headers'
-- Conan: Component target declared 'Boost::boost'
-- Conan: Component target declared 'boost::_libboost'
-- Conan: Component target declared 'Boost::atomic'
-- Conan: Component target declared 'Boost::container'
-- Conan: Component target declared 'Boost::context'
-- Conan: Component target declared 'Boost::date_time'
-- Conan: Component target declared 'Boost::exception'
-- Conan: Component target declared 'Boost::math'
-- Conan: Component target declared 'Boost::math_c99'
-- Conan: Component target declared 'Boost::math_c99f'
-- Conan: Component target declared 'Boost::math_c99l'
-- Conan: Component target declared 'Boost::math_tr1'
-- Conan: Component target declared 'Boost::math_tr1f'
-- Conan: Component target declared 'Boost::math_tr1l'
-- Conan: Component target declared 'Boost::program_options'
-- Conan: Component target declared 'Boost::regex'
-- Conan: Component target declared 'Boost::serialization'
-- Conan: Component target declared 'Boost::stacktrace'
-- Conan: Component target declared 'Boost::stacktrace_addr2line'
-- Conan: Component target declared 'Boost::stacktrace_backtrace'
-- Conan: Component target declared 'Boost::stacktrace_basic'
-- Conan: Component target declared 'Boost::stacktrace_noop'
-- Conan: Component target declared 'Boost::system'
-- Conan: Component target declared 'Boost::test'
-- Conan: Component target declared 'Boost::test_exec_monitor'
-- Conan: Component target declared 'Boost::url'
-- Conan: Component target declared 'Boost::wserialization'
-- Conan: Component target declared 'Boost::chrono'
-- Conan: Component target declared 'Boost::coroutine'
-- Conan: Component target declared 'Boost::filesystem'
-- Conan: Component target declared 'Boost::json'
-- Conan: Component target declared 'Boost::nowide'
-- Conan: Component target declared 'Boost::prg_exec_monitor'
-- Conan: Component target declared 'Boost::random'
-- Conan: Component target declared 'Boost::thread'
-- Conan: Component target declared 'Boost::timer'
-- Conan: Component target declared 'Boost::type_erasure'
-- Conan: Component target declared 'Boost::unit_test_framework'
-- Conan: Component target declared 'Boost::wave'
-- Conan: Component target declared 'Boost::contract'
-- Conan: Component target declared 'Boost::fiber'
-- Conan: Component target declared 'Boost::fiber_numa'
-- Conan: Component target declared 'Boost::graph'
-- Conan: Component target declared 'Boost::iostreams'
-- Conan: Component target declared 'Boost::locale'
-- Conan: Component target declared 'Boost::log'
-- Conan: Component target declared 'Boost::log_setup'
-- Conan: Target declared 'boost::boost'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/home/achim/.conan/data/bzip2/1.0.8/_/_/package/c32092bf4d4bb47cf962af898e02823f499b017e/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'libbacktrace::libbacktrace'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release
foxglove-websocket/1.1.0 (test package): CMake command: cmake --build "/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release" '--' '-j16'
Consolidate compiler generated dependencies of target test_package
[ 50%] Building CXX object CMakeFiles/test_package.dir/test_package.cpp.o
[100%] Linking CXX executable test_package
[100%] Built target test_package
foxglove-websocket/1.1.0 (test package): Running test()</pre></details>


<details>
<summary>Build logs for -o foxglove-websocket:asio=False</summary>
<pre>achim@achim-pc:~/workspace/conan-center-index/recipes/foxglove-websocket$ conan create all/conanfile.py foxglove-websocket/1.1.0@ -pr:b=default -pr:h=default -o foxglove-websocket:asio=False
[HOOK - conan-center.py] pre_export(): [DEPRECATED GLOBAL CPPSTD (KB-H001)] OK
[HOOK - conan-center.py] pre_export(): [REFERENCE LOWERCASE (KB-H002)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE METADATA (KB-H003)] OK
[HOOK - conan-center.py] pre_export(): [HEADER_ONLY, NO COPY SOURCE (KB-H005)] OK
[HOOK - conan-center.py] pre_export(): [FPIC OPTION (KB-H006)] OK
[HOOK - conan-center.py] pre_export(): [VERSION RANGES (KB-H008)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] Total recipe size: 6.1044921875 KB
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] OK
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: None
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: None
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE FOLDER (KB-H024)] OK
[HOOK - conan-center.py] pre_export(): [META LINES (KB-H025)] OK
[HOOK - conan-center.py] pre_export(): [CONAN CENTER INDEX URL (KB-H027)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE MINIMUM VERSION (KB-H028)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - RUN ENVIRONMENT (KB-H029)] OK
[HOOK - conan-center.py] pre_export(): [SYSTEM REQUIREMENTS (KB-H032)] OK
[HOOK - conan-center.py] pre_export(): [CONANDATA.YML FORMAT (KB-H030)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - NO IMPORTS() (KB-H034)] OK
[HOOK - conan-center.py] pre_export(): [NO AUTHOR (KB-H037)] OK
[HOOK - conan-center.py] pre_export(): [NOT ALLOWED ATTRIBUTES (KB-H039)] OK
[HOOK - conan-center.py] pre_export(): [NO TARGET NAME (KB-H040)] OK
[HOOK - conan-center.py] pre_export(): [NO FINAL ENDLINE (KB-H041)] OK
[HOOK - conan-center.py] pre_export(): [NO REQUIRES.ADD() (KB-H044)] OK
[HOOK - conan-center.py] pre_export(): [DELETE OPTIONS (KB-H045)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERBOSE MAKEFILE (KB-H046)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERSION REQUIRED (KB-H048)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE WINDOWS EXPORT ALL SYMBOLS (KB-H049)] OK
[HOOK - conan-center.py] pre_export(): [DEFAULT OPTIONS AS DICTIONARY (KB-H051)] OK
[HOOK - conan-center.py] pre_export(): [CONFIG.YML HAS NEW VERSION (KB-H052)] OK
[HOOK - conan-center.py] pre_export(): [PRIVATE IMPORTS (KB-H053)] OK
[HOOK - conan-center.py] pre_export(): [SINGLE REQUIRES (KB-H055)] OK
[HOOK - conan-center.py] pre_export(): [TOOLS RENAME (KB-H057)] OK
[HOOK - conan-center.py] pre_export(): [ILLEGAL CHARACTERS (KB-H058)] OK
[HOOK - conan-center.py] pre_export(): [CLASS NAME (KB-H059)] OK
[HOOK - conan-center.py] pre_export(): [NO CRLF (KB-H060)] OK
[HOOK - conan-center.py] pre_export(): [NO BUILD SYSTEM FUNCTIONS (KB-H061)] OK
[HOOK - conan-center.py] pre_export(): [TOOLS CROSS BUILDING (KB-H062)] OK
[HOOK - conan-center.py] pre_export(): [INVALID TOPICS (KB-H064)] OK
[HOOK - conan-center.py] pre_export(): [NO REQUIRED_CONAN_VERSION (KB-H065)] OK
[HOOK - conan-center.py] pre_export(): [TEST_TYPE MANAGEMENT (KB-H068)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - NO DEFAULT OPTIONS (KB-H069)] OK
[HOOK - conan-center.py] pre_export(): [MANDATORY SETTINGS (KB-H070)] OK
[HOOK - conan-center.py] pre_export(): [PYLINT EXECUTION (KB-H072)] OK
[HOOK - conan-center.py] pre_export(): [TEST V1 PACKAGE FOLDER (KB-H073)] OK
[HOOK - conan-center.py] pre_export(): [REQUIREMENT OVERRIDE PARAMETER (KB-H075)] OK
Exporting package recipe
foxglove-websocket/1.1.0 exports: File 'conandata.yml' found. Exporting it...
foxglove-websocket/1.1.0 exports: Copied 1 '.yml' file: conandata.yml
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] Saving conandata.yml: {'sources': {'1.1.0': {'url': 'https://github.com/foxglove/ws-protocol/archive/refs/tags/releases/cpp/v1.1.0.tar.gz', 'sha256': '360c246f5edccbe2a7c905ff77c5fb2b8ca723e6b441613871952d9d19903362'}}}
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] New conandata.yml contents: sources:
  1.1.0:
    sha256: 360c246f5edccbe2a7c905ff77c5fb2b8ca723e6b441613871952d9d19903362
    url: https://github.com/foxglove/ws-protocol/archive/refs/tags/releases/cpp/v1.1.0.tar.gz

[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] OK
[HOOK - conan-center.py] post_export(): [DEFAULT SHARED OPTION VALUE (KB-H050)] OK
[HOOK - conan-center.py] post_export(): [LICENSE PUBLIC DOMAIN (KB-H056)] OK
foxglove-websocket/1.1.0: The stored package has not changed
foxglove-websocket/1.1.0: Source folder is corrupted, forcing removal
foxglove-websocket/1.1.0: Exported revision: 067f8793cbc242cc56a383a86713f311
Configuration (profile_host):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
foxglove-websocket:asio=False
[build_requires]
[env]

Configuration (profile_build):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

foxglove-websocket/1.1.0: Forced build from source
Version ranges solved
    Version range '>=1.1 <4' required by 'websocketpp/0.8.2' resolved to 'openssl/3.1.0' in local cache

foxglove-websocket/1.1.0 (test package): Installing package
Requirements
    foxglove-websocket/1.1.0 from local cache - Cache
    nlohmann_json/3.10.5 from 'conancenter' - Cache
    openssl/3.1.0 from 'conancenter' - Cache
    websocketpp/0.8.2 from 'conancenter' - Cache
    zlib/1.2.13 from 'conancenter' - Cache
Packages
    foxglove-websocket/1.1.0:a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf - Build
    nlohmann_json/3.10.5:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    openssl/3.1.0:304480252b01879c8641f79a653b593b8f26cf9f - Cache
    websocketpp/0.8.2:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    zlib/1.2.13:dfbe50feef7f3c6223a476cd5aeadb687084a646 - Cache

Installing (downloading, building) binaries...
nlohmann_json/3.10.5: Already installed!
zlib/1.2.13: Already installed!
openssl/3.1.0: Already installed!
websocketpp/0.8.2: Already installed!
foxglove-websocket/1.1.0: WARN: Build folder is dirty, removing it: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf
[HOOK - conan-center.py] pre_source(): [IMMUTABLE SOURCES (KB-H010)] OK
foxglove-websocket/1.1.0: Configuring sources in /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/source/src
Downloading v1.1.0.tar.gz completed [170.78k]                                            foxglove-websocket/1.1.0: t/1.1.0: 
foxglove-websocket/1.1.0: 
foxglove-websocket/1.1.0: Copied 1 '.py' file: conanfile.py
foxglove-websocket/1.1.0: Copied 1 file: LICENSE
foxglove-websocket/1.1.0: Copied 1 '.txt' file: CMakeLists.txt
foxglove-websocket/1.1.0: Copied 3 '.cpp' files: serialization.cpp, base64.cpp, parameter.cpp
foxglove-websocket/1.1.0: Copied 12 '.hpp' files
[HOOK - conan-center.py] post_source(): [LIBCXX MANAGEMENT (KB-H011)] OK
[HOOK - conan-center.py] post_source(): [CPPSTD MANAGEMENT (KB-H022)] OK
[HOOK - conan-center.py] post_source(): [SHORT_PATHS USAGE (KB-H066)] OK
foxglove-websocket/1.1.0: Copying sources to build folder
foxglove-websocket/1.1.0: Building your package in /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf
foxglove-websocket/1.1.0: Generator 'CMakeDeps' calling 'generate()'
foxglove-websocket/1.1.0: Calling generate()
foxglove-websocket/1.1.0: Preset 'release' added to CMakePresets.json. Invoke it manually using 'cmake --preset release'
foxglove-websocket/1.1.0: If your CMake version is not compatible with CMakePresets (<3.19) call cmake like: 'cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release/generators/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release'
foxglove-websocket/1.1.0: Aggregating env generators
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK. 'fPIC' option found and apparently well managed
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK
foxglove-websocket/1.1.0: Calling build()
foxglove-websocket/1.1.0: CMake command: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/src"
-- Using Conan toolchain: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The CXX compiler identification is GNU 11.3.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Target declared 'nlohmann_json::nlohmann_json'
-- Conan: Target declared 'websocketpp::websocketpp'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/achim/.conan/data/openssl/3.1.0/_/_/package/304480252b01879c8641f79a653b593b8f26cf9f/lib/cmake/conan-official-openssl-variables.cmake'
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_POLICY_DEFAULT_CMP0091


-- Build files have been written to: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release
foxglove-websocket/1.1.0: CMake command: cmake --build "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release" '--' '-j16'
[ 75%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/base64.cpp.o
[ 75%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/serialization.cpp.o
[ 75%] Building CXX object CMakeFiles/foxglove_websocket.dir/src/parameter.cpp.o
[100%] Linking CXX static library libfoxglove_websocket.a
[100%] Built target foxglove_websocket
foxglove-websocket/1.1.0: Package 'a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf' built
foxglove-websocket/1.1.0: Build folder /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release
foxglove-websocket/1.1.0: Generated conaninfo.txt
foxglove-websocket/1.1.0: Generated conanbuildinfo.txt
foxglove-websocket/1.1.0: Generating the package
foxglove-websocket/1.1.0: Package folder /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf
foxglove-websocket/1.1.0: Calling package()
foxglove-websocket/1.1.0: CMake command: cmake --install "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/build/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/build/Release" --prefix "/home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf"
-- Install configuration: "Release"
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/lib/libfoxglove_websocket.a
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/base64.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/server_interface.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_tls.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_client.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_logging.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_server.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/callback_queue.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/regex_utils.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_notls.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/serialization.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/parameter.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/common.hpp
-- Installing: /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/licenses/LICENSE
[HOOK - conan-center.py] post_package(): [PACKAGE LICENSE (KB-H012)] OK
[HOOK - conan-center.py] post_package(): [DEFAULT PACKAGE LAYOUT (KB-H013)] OK
[HOOK - conan-center.py] post_package(): [MATCHING CONFIGURATION (KB-H014)] OK
[HOOK - conan-center.py] post_package(): [SHARED ARTIFACTS (KB-H015)] OK
[HOOK - conan-center.py] post_package(): [STATIC ARTIFACTS (KB-H074)] OK
[HOOK - conan-center.py] post_package(): [EITHER STATIC OR SHARED OF EACH LIB (KB-H076)] OK
[HOOK - conan-center.py] post_package(): [PC-FILES (KB-H020)] OK
[HOOK - conan-center.py] post_package(): [CMAKE-MODULES-CONFIG-FILES (KB-H016)] OK
[HOOK - conan-center.py] post_package(): [PDB FILES NOT ALLOWED (KB-H017)] OK
[HOOK - conan-center.py] post_package(): [LIBTOOL FILES PRESENCE (KB-H018)] OK
[HOOK - conan-center.py] post_package(): [MS RUNTIME FILES (KB-H021)] OK
[HOOK - conan-center.py] post_package(): [SHORT_PATHS USAGE (KB-H066)] OK
[HOOK - conan-center.py] post_package(): [MISSING SYSTEM LIBS (KB-H043)] OK
[HOOK - conan-center.py] post_package(): [APPLE RELOCATABLE SHARED LIBS (KB-H077)] OK
foxglove-websocket/1.1.0 package(): Packaged 1 file: LICENSE
foxglove-websocket/1.1.0 package(): Packaged 1 '.a' file: libfoxglove_websocket.a
foxglove-websocket/1.1.0 package(): Packaged 12 '.hpp' files
foxglove-websocket/1.1.0: Package 'a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf' created
foxglove-websocket/1.1.0: Created package revision 0dc2f01deef53f7c236bc8f9a73f90fa
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
[HOOK - conan-center.py] post_package_info(): [LIBRARY DOES NOT EXIST (KB-H054)] OK
[HOOK - conan-center.py] post_package_info(): [INCLUDE PATH DOES NOT EXIST (KB-H071)] OK
foxglove-websocket/1.1.0 (test package): Generator 'CMakeDeps' calling 'generate()'
foxglove-websocket/1.1.0 (test package): Generator txt created conanbuildinfo.txt
foxglove-websocket/1.1.0 (test package): Calling generate()
foxglove-websocket/1.1.0 (test package): Preset 'release' added to CMakePresets.json. Invoke it manually using 'cmake --preset release'
foxglove-websocket/1.1.0 (test package): If your CMake version is not compatible with CMakePresets (<3.19) call cmake like: 'cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release'
foxglove-websocket/1.1.0 (test package): Aggregating env generators
foxglove-websocket/1.1.0 (test package): Generated conaninfo.txt
foxglove-websocket/1.1.0 (test package): Generated graphinfo
Using lockfile: '/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan.lock'
Using cached profile from lockfile
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not found
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK
foxglove-websocket/1.1.0 (test package): Calling build()
foxglove-websocket/1.1.0 (test package): CMake command: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/."
-- Using Conan toolchain: /home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release/generators/conan_toolchain.cmake
-- Conan: Target declared 'foxglove-websocket::foxglove-websocket'
-- Conan: Target declared 'nlohmann_json::nlohmann_json'
-- Conan: Target declared 'websocketpp::websocketpp'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/achim/.conan/data/openssl/3.1.0/_/_/package/304480252b01879c8641f79a653b593b8f26cf9f/lib/cmake/conan-official-openssl-variables.cmake'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release
foxglove-websocket/1.1.0 (test package): CMake command: cmake --build "/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release" '--' '-j16'
Consolidate compiler generated dependencies of target test_package
[ 50%] Building CXX object CMakeFiles/test_package.dir/test_package.cpp.o
In file included from /home/achim/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/websocketpp/transport/asio/base.hpp:31,
                 from /home/achim/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/websocketpp/transport/asio/connection.hpp:31,
                 from /home/achim/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/websocketpp/transport/asio/endpoint.hpp:32,
                 from /home/achim/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/websocketpp/config/asio_no_tls.hpp:32,
                 from /home/achim/.conan/data/foxglove-websocket/1.1.0/_/_/package/a6ce6a6ef00edf7459e04e5d83236d365d0b6fdf/include/foxglove/websocket/websocket_notls.hpp:3,
                 from /home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/test_package.cpp:1:
/home/achim/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/websocketpp/common/asio.hpp:56:14: fatal error: boost/version.hpp: No such file or directory
   56 |     #include <boost/version.hpp>
      |              ^~~~~~~~~~~~~~~~~~~
compilation terminated.
gmake[2]: *** [CMakeFiles/test_package.dir/build.make:76: CMakeFiles/test_package.dir/test_package.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/test_package.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
ERROR: foxglove-websocket/1.1.0 (test package): Error in build() method, line 22
        cmake.build()
        ConanException: Error 2 while executing cmake --build "/home/achim/workspace/conan-center-index/recipes/foxglove-websocket/all/test_package/build/Release" '--' '-j16'</pre></details>